build.cmd

```diff
@@ -1,3 +1,10 @@
 @echo off
+set CACHEDMSBUILDDEBUGONSTART=%MSBUILDDEBUGONSTART%
+if %MSBUILDDEBUGONSTART% == 0 goto build
+set /p ans=Did you mean to have MSBUILDDEBUGONSTART equal %MSBUILDDEBUGONSTART%? (y/n) 
+if %ans% == y goto build
+set MSBUILDDEBUGONSTART=0
```

Comment:
```suggestion
set MSBUILDDEBUGONSTART=
```

I prefer unsetting because we're not super consistent about which of our environment variables get "if the environment variable is set to anything at all" versus "set to specific value" treatment.